schema-admin.graphql

```diff
@@ -487,6 +487,97 @@ input CreateApprovedCuratedCorpusItemInput {
     scheduledSurfaceGuid: ID
 }
 
+"""
+Input data for loading an Approved Item via an automated process and optionally scheduling
+this item to appear on a Scheduled Surface.
+"""
+input LoadApprovedCuratedCorpusItemInput {
+    """
+    The GUID of the corresponding Prospect ID. Will be empty for manually added items.
+    """
+    prospectId: ID
+    """
+    The URL of the Approved Item.
+    """
+    url: Url!
+    """
+    The title of the Approved Item.
+    """
+    title: String!
+    """
+    The excerpt of the Approved Item.
+    """
+    excerpt: String!
+    """
+    The outcome of the curators' review of the Approved Item.
+    """
+    status: CuratedStatus!
+    """
+    What language this item is in. This is a two-letter code, for example, 'en' for English.
+    """
+    language: String!
+    """
+    The name of the online publication that published this story.
+    """
+    publisher: String!
+    """
+    The image URL for this item's accompanying picture.
+    """
+    imageUrl: Url!
+    """
+    A topic this story best fits in.
+    Temporarily a string value that will be provided by Prospect API, possibly an enum in the future.
+    """
+    topic: String!
+    """
+    Whether this story is a Pocket Collection.
+    """
+    isCollection: Boolean
+    """
+    A flag to ML to not recommend this item long term after it is added to the corpus.
+    Example: a story covering an election, or "The best of 202x" collection.
+    """
+    isTimeSensitive: Boolean
+    """
+    Whether this item is a syndicated article.
+    """
+    isSyndicated: Boolean
+    """
+    A Unix timestamp of when the entity was created.
+    """
+    createdAt: Int!
+    """
+    A single sign-on user identifier of the user who created this entity.
+    """
+    createdBy: String!
```

Comment:
Maybe I missed it but do we have a `createdBy` for the entries/items that will be migrated? 

cc @sri-kirsh

Reply:
we have `createdBy` field but they are not always SSO username

Reply:
update, we have only 10 curators and only few of them are non-sso names. we can individually map them 
https://pocket.slack.com/archives/CTLKW5WCF/p1646848666279309?thread_ts=1646847427.826849&cid=CTLKW5WCF